translations/ui-orders/en.json

```diff
@@ -138,7 +138,7 @@
   "errors.currentFYearNotFound": "Current fiscal year not found for ledger",
   "errors.electronicLocCostQtyMismatch": "PO line electronic quantity and locations electronic quantity do not match",
   "errors.encumbranceCreationFailure": "One or more encumbrance record(s) failed to be created",
-  "errors.fundCannotBePaid": "One or more fund distributions on this order can not be encumbered, because there is not enough money in the budget",
+  "errors.fundCannotBePaid": "One or more fund distributions on this order can not be encumbered, because there is not enough money in <b>{fundCodes}</b>.",
```

Comment:
@dltj could you please update translations? thanks!

Reply:
Done: https://github.com/folio-org/ui-orders/pull/1328

Reply:
Unfortunately, the key `errors.fundCannotBePaid` is not updated

Reply:
Hmmm...Lokalise did not have the latest code.  I forced a refresh on Lokalise and started the process again: https://github.com/folio-org/ui-orders/pull/1329